app.js

```diff
@@ -4,14 +4,18 @@
 var baseurl = process.env.SUBFOLDER || '/';
 var app = require('express')();
 var { DownloaderHelper } = require('node-downloader-helper');
+const { HttpProxyAgent } = require('http-proxy-agent');
+const { HttpsProxyAgent } = require('https-proxy-agent');
+const { URL } = require('url');
+var getProxyForUrl = require('proxy-from-env').getProxyForUrl;
 var exec = require('child_process').exec;
 var express = require('express');
 var fs = require('fs');
 var http = require('http').Server(app);
 var io = require('socket.io')(http, {path: baseurl + 'socket.io'});
 var isBinaryFile = require("isbinaryfile").isBinaryFile;
 var path = require('path');
-var readdirp = require('readdirp');
+let {readdirp} = require('readdirp');
```

Comment:
Basic approach and language was changed in the `readdirp` starting with `v4.0.0`.